src/screens/CompanyScreen.js

```diff
@@ -67,6 +69,8 @@ export const CompanyScreen = ({ navigation }) => {
   const { sections = {} } = globalSettings;
   const { headlineCompany = texts.homeTitles.company } = sections;
 
+  useMatomoTrackScreenView(MATOMO_TRACKING.SCREEN_VIEW.COMPANY);
```

Comment:
could be moved up as well

Reply:
done in https://github.com/ikuseiGmbH/smart-village-app-app/pull/109/commits/f22a1566b20d2eded43188bff5bc48e012167c41